index.js

```diff
@@ -219,7 +219,7 @@ module.exports = (async() => {
       },
     },
     security: [{ 'Bearer': []}],
-    jsonEditor: true,
+    // jsonEditor: true,
```

Comment:
I recently enabled the jsonEditor because it looked like it made it easier to compose requests, however it is too limiting and forces the presence of all possible fields, which conflicts with our data models. Back to the json text area we go.